src/PhpGenerator/Method.php

```diff
@@ -81,9 +88,10 @@ public static function from($from)
 	public function __toString()
 	{
 		$parameters = [];
+		static $builtinTypes = ['array', 'string', 'bool', 'float', 'int', 'callable', ''];
```

Comment:
Well, `self` and `parent` [return false](https://3v4l.org/NABuu) when checked by `ReflectionType::isBuiltin()`.